modules/opensimplex/noise_texture.h

```diff
@@ -56,6 +56,7 @@ class NoiseTexture : public Texture2D {
 
 	Ref<OpenSimplexNoise> noise;
 	Vector2i size = Vector2i(512, 512);
+	Vector2 noise_offset = Vector2(0, 0);
```

Comment:
```suggestion
	Vector2 noise_offset;
```
`Vector2` is constructed with `0, 0` values out of the box.